src/mpyl/cli/build.py

```diff
@@ -116,8 +125,24 @@ def build(ctx, config, properties, verbose):
     help="Build all projects, regardless of changes on branch",
 )
 @click.option("--tag", "-t", help="Tag to build", type=click.STRING, required=False)
+@click.option(
+    "--stage",
+    default=None,
+    type=click.Choice([stage.value for stage in Stage]),
+    required=False,
+    help="Stage to run",
+)
+@click.option(
+    "--sequential",
```

Comment:
Not 100% sure about the naming, but could be good. If I'm thinking CI, it's more like we need to run this stage sequentially (and not in parallel). Could be maybe `--continued` or something?

Reply:
I don't have a strong opinion either way, we can let @SamTheisens decide which option he prefers

Reply:
I'm not entirely clear in which scenarios you would _not_ want to append results to previous ones?

Reply:
In case you're running the first stage of a run and the results of a previous run are still in the workspace. Related to the discussion we had on whatsapp about not assuming the results are cleared already.
The only other solution I see is deleting the results in the jenkinsfile, which is not ideal either.